Gulpfile.js

```diff
@@ -22,7 +22,15 @@ gulp.task("build", function () {
     }))
     .pipe(through.obj(function (file, enc, callback) {
       file._path = file.path;
-      file.path = file.path.replace(/^([^\\]+)\/src/, "$1/lib");
+      // Replace src with lib
+      var sub = file.path.substr(__dirname.length);
```

Comment:
Only with an ES6 Gulpfile and only if you wanted to replace *all* instances of src.


-----Original Message-----
From: "James Kyle" <notifications@github.com>
Sent: ‎12/‎12/‎2015 9:33 PM
To: "babel/babel" <babel@noreply.github.com>
Cc: "Tim De Pauw" <tim@pwnt.be>
Subject: Re: [babel] Fixing some Windows build issues (#3125)

In Gulpfile.js:
> @@ -22,7 +22,15 @@ gulp.task("build", function () {
>      }))
>      .pipe(through.obj(function (file, enc, callback) {
>        file._path = file.path;
> -      file.path = file.path.replace(/^([^\\]+)\/src/, "$1/lib");
> +      // Replace src with lib
> +      var sub = file.path.substr(__dirname.length);
You could make this a little easier to follow by doing this:
path.normalize(file.path)
  .split(path.sep)
  .map(n => n === 'src' ? 'lib' : n) 
  .join(path.sep);—
Reply to this email directly or view it on GitHub.